string[] joint_names
float32[] joint_values
---
bool success
